src/Symfony/Component/DependencyInjection/Compiler/ResolveInstanceofConditionalsPass.php

```diff
@@ -64,9 +64,10 @@ private function processDefinition(ContainerBuilder $container, $id, Definition
         $definition->setInstanceofConditionals([]);
         $parent = $shared = null;
         $instanceofTags = [];
+        $reflectionClass = $container->getReflectionClass($class, false);
```

Comment:
this calls the method all the time while before this was behind a condition
we need to preserve this conditional call the first time, this will save some resource tracking